shapefile.go

```diff
@@ -12,19 +12,19 @@ type ShapeType int32
 // These are the possible shape types.
```

Comment:
It would be neat if you embedded a `go:generate` instruction in `shapefile.go` with the call to `stringer` that you used.

Reply:
@fawick done